### DISCUSSION TITLE
MMS new message with Wireless connexion only ?

### DISCUSSION THREAD

**Discussion by caywin25** [FossifyOrg/General-Discussion#508] [Q&A]
Hello and thank you for your work.

I am often with wifi on the phone (the mobile connection is disabled)
When someone sends me an MMS I have an android notification telling me that I have received an MMS and that I must retrieve it with the data connection but in your Messages application I have no notification of any new message and I lose the MMS, I cannot retrieve it. 

Is it possible that when I am just with my Wifi connection and someone sends me an MMS, the Messages application displays a new message from the person in question with the MMS?

I don't know if this is very clear as an explanation :)

Thank you

**Comment by naveensingh** [comment]
Sorry for not responding sooner. This is indeed a bug, the app should at least download the message when mobile data is available. 

For future updates: https://github.com/FossifyOrg/Messages/issues/298
